test/sharness/t0121-bootstrap-iptb.sh

```diff
@@ -0,0 +1,65 @@
+#!/bin/sh
+#
+# Copyright (c) 2016 Jeromy Johnson
```

Comment:
i stopped and admired the fact that the script i copied this from was 2014. I had to bump it two years forward